.github/workflows/log_viewer.yml

```diff
@@ -20,11 +20,45 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: "22.x"
+
       - name: Install dependencies
         run: yarn install
+
+      - name: Cache node_modules
```

Comment:
I added this because "node_modules are slow" and we've got a load of `yarn installs` here, but when I tested it I think this actually slows down the build when the package.lock has changed. It currently takes about 5s to yarn install because we have so few dependencies, but *saving* the cache takes a [couple of minutes](https://github.com/UKGovernmentBEIS/inspect_ai/actions/runs/10219779258/job/28278735407?pr=183) 🙄 

Want me to remove the cache stuff or is it fine to get it in now before the inevitable dependency growth? Happy either way 👍

Reply:
Yeah let's remove the cache stuff for now.